src/components/atoms/CardInfoWithCount/CardInfoWithCount.tsx

```diff
@@ -0,0 +1,38 @@
+import * as styles from "./CardInfoWithCount.css";
+
+import { ThumbsUp, ThumbsDown, MessageSquare } from "react-feather";
+
+export const CardInfoWithCount: React.FC<{
+  writer: string;
+  like: number;
+  dislike: number;
+  comment: number;
+}> = ({ writer, like, dislike, comment }) => {
+  return (
+    <div className={styles.contentInfo}>
+      <div className={styles.infoes}>
+        <div className={styles.writer}>{writer}</div>
+      </div>
```

Comment:
`writer` 하나만 있어서 `styles.infoes`를 적용시켜주지 않아도 될 것 같습니다! 그리고 여기도 마찬가지로 `span`이 더 맞는 것 같아용
```suggestion
        <div className={styles.writer}>{writer}</div>
```